build-aux/main.mk

```diff
@@ -334,7 +335,7 @@ endif
 	# We run the test suite with TELEPRESENCE_LOGIN_DOMAIN set to localhost since that value
 	# is only used for extensions. Therefore, we want to validate that our tests, and
 	# telepresence, run without requiring any outside dependencies.
-	TELEPRESENCE_MAX_LOGFILES=300 TELEPRESENCE_LOGIN_DOMAIN=127.0.0.1 CGO_ENABLED=$(CGO_ENABLED) go test -v -timeout=55m ./integration_test/...
```

Comment:
i've decided that this change should be merged if the reviewers agree with it :) i'm sick of waiting an hour+ to see tests fail. I think if they're gonna die let them die early from now on